documentation/tutorial.md

```diff
@@ -42,7 +42,7 @@ You can list all LXD containers by entering the command `lxc list` in to the com
 
 
 ## Install and prepare Juju
-Juju is an Operator Lifecycle Manager (OLM) for clouds, bare metal, LXD or Kubernetes. We will be using it to deploy and manage Charmed MongoDB. As with LXD, Juju is installed from a snap package:
+[Juju](https://juju.is/) is an Operator Lifecycle Manager (OLM) for clouds, bare metal, LXD or Kubernetes. We will be using it to deploy and manage Charmed MongoDB. As with LXD, Juju is installed from a snap package:
```

Comment:
juju is already linked on line 2. I initially had this link here too, but docs team said it would be best to have just one.

Reply:
Reverted.